vars/tvm-ci-prod.auto.tfvars

```diff
@@ -87,7 +87,7 @@ persistent_agent_types = {
 autoscaler_types = {
   "Prod-Autoscaler-Jenkins-CPU" = {
     image_family        = "jenkins-stock-agent"
-    agent_instance_type = "r5.large"
+    agent_instance_type = "c4.xlarge"
```

Comment:
should this be a bigger type (like 2x or 4xlarge)? I'm worried that we'll hit OOM issues otherwise

Reply:
ah I was trying to use the same node type as the autoscalers and missed the `4`, fixed